.drone.jsonnet

```diff
@@ -0,0 +1,43 @@
+local tags = ['v3.6', 'v3.7', 'v3.8', 'v3.9', 'edge'];
+local tags_armv7 = ['v3.9', 'edge'];
+
+local pipeline(arch, darch, tags) = {
+  kind: 'pipeline',
+  name: arch,
+  platform: {
+    os: 'linux',
+    arch: darch,
+  },
+  steps: [
+    {
+      name: 'dockerfiles',
+      image: 'alpine',
+      commands: ['./dockerfiles.sh'],
+    },
+  ] + [
+    {
+      name: tag,
+      image: 'plugins/docker',
+      settings: {
+        username: {
+          from_secret: 'docker_user',
+        },
+        password: {
+          from_secret: 'docker_pass',
+        },
+        repo: 'clandmeter/docker-abuild',
+        tags: '%s-%s' % [std.strReplace(tag, 'v', ''), arch],
+        dockerfile: 'Dockerfiles/%s/%s/Dockerfile' % [tag, arch],
+      },
```

Comment:
```suggestion
      },
      depends_on: ["dockerfiles"],
```

Reply:
depends_on provides parallel execution?

Reply:
Yes. https://discourse.drone.io/t/how-to-setup-parallel-pipeline-steps-1-0/3251

Reply:
Very nice, added.